docs/nodes/nodes-library/nodes/Telegram/README.md

```diff
@@ -0,0 +1,55 @@
+---
+permalink: /nodes/n8n-nodes-base.telegram
+---
+
+# Telegram
+
+[Telegram](https://telegram.org) is a cloud-based instant messaging and voice-over-IP service.
+
+::: tip 🔑 Credentials
+You can find authentication information for this node [here](../../../credentials/Telegram/README.md).
+:::
+
+## Basic Operations
+
+- Chat
+	- Get up-to-date information about a chat
```

Comment:
<img width="271" alt="Screenshot 2020-07-09 at 09 39 44" src="https://user-images.githubusercontent.com/7481165/87011386-2238f700-c1c8-11ea-8f1e-3e00a52810b0.png">

Reply:
The hyphens need to be removed.